### PR TITLE
Connect logging hierarchy 2

### PR DESCRIPTION
**Pull request by tombentley**
Alternative solution to https://github.com/strimzi/strimzi-kafka-operator/pull/5138

### CODE REVIEW

File: cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiImpl.java [hunk 1]
```diff
@@ -5,6 +5,17 @@
 
 package io.strimzi.operator.cluster.operator.assembly;
 
+import java.io.IOException;
```
Comment (sknot-rh) [inline cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiImpl.java:8]: I see you have some import relocation. is it intended?

**Review by sknot-rh — APPROVED** [review]
LGTM

**Review by scholzj — APPROVED** [review]

**Comment by scholzj** [comment]
Is this OK to merge? Or do you plan to do something with the imports?

**Pull request merged**